lib/open_mic.rb

```diff
@@ -0,0 +1,15 @@
+require 'pry'
+class OpenMic
+
+attr_reader :name, :date, :welcome
+
+  def initialize (name)
```

Comment:
I think the key here is that we are not initializing with a 'name', but with a hash of information, which we can then use to grab specific attributes:

```
def initialize(open_mic_info)
  @location = open_mic_info[:location]
  @date = open_mic_info[:date]
end
```